pkg/resources/row_access_policy_acceptance_test.go

```diff
@@ -404,29 +475,48 @@ func TestAcc_view_migrateFromVersion_0_95_0(t *testing.T) {
 					},
 				},
 				Config: rowAccessPolicy_v0_95_0(id, body),
+				ConfigPlanChecks: resource.ConfigPlanChecks{
+					PostApplyPostRefresh: []plancheck.PlanCheck{
+						// expect change - arg name is lower case which causes a diff
+						plancheck.ExpectResourceAction(resourceName, plancheck.ResourceActionDestroyBeforeCreate),
+					},
+				},
+				ExpectNonEmptyPlan: true,
 				Check: assert.AssertThat(t, resourceassert.RowAccessPolicyResource(t, resourceName).
 					HasNameString(id.Name()).
 					HasDatabaseString(id.DatabaseName()).
 					HasSchemaString(id.SchemaName()).
 					HasFullyQualifiedNameString(id.FullyQualifiedName()),
 					assert.Check(resource.TestCheckResourceAttr(resourceName, "row_access_expression", body)),
+					assert.Check(resource.TestCheckResourceAttr(resourceName, "signature.A", "VARCHAR")),
+					assert.Check(resource.TestCheckResourceAttr(resourceName, "signature.B", "VARCHAR")),
```

Comment:
Why it's lower case in config, here it's with upper case and in the latest version it's with lower case? In the state upgrader you upper case'd it, so it should be `B` right ?